app/src/main/java/woowacourse/omok/omokdb/OmokDao.kt

```diff
@@ -0,0 +1,64 @@
+package woowacourse.omok.omokdb
+
+import android.content.Context
+import android.database.Cursor
+import android.provider.BaseColumns
+import androidx.core.content.contentValuesOf
+
+class OmokDao(context: Context) {
+    private val dbHelper = OmokDbHelper(context)
+
+    fun insert(omokEntry: OmokEntry): OmokEntry {
+        val db = dbHelper.writableDatabase
+        val id =
+            db.insert(
+                OmokContract.TABLE_NAME,
+                null,
+                contentValuesOf(
+                    OmokContract.COLUMN_ROW to omokEntry.row,
+                    OmokContract.COLUMN_COLUMN to omokEntry.column,
+                    OmokContract.COLUMN_COLOR to omokEntry.color,
+                ),
+            )
+        return omokEntry.copy(id = id)
```

Comment:
```suggestion
    fun insert(omokEntry: OmokEntry): OmokEntry {
        val db = dbHelper.writableDatabase
        val id =
            db.insert(
                OmokContract.TABLE_NAME,
                null,
                contentValuesOf(
                    OmokContract.COLUMN_ROW to omokEntry.column,
                    OmokContract.COLUMN_COLUMN to omokEntry.row,
                    OmokContract.COLUMN_COLOR to omokEntry.color,
                ),
            )
        return omokEntry.copy(id = id)
```

누군가 실수로 row 와 column 을 insert 쿼리를 짤 때 헷갈리면 어떻게 될까요?